sbws/util/config.py

```diff
@@ -70,7 +70,8 @@ def _get_user_config(args, conf=None):
             # sbws should start with a logger before reading configurations.
             print('Configuration file %s not found, using defaults.' %
                   args.config)
-        return _extend_config(conf, args.config)
+        else:
```

Comment:
i think it would be more clear if instead of else you return _extend_config here
```suggestion
        return _extend_config(conf, args.config)
```

Reply:
It would also cause us to try to read a file that doesn't exist.

Reply:
No when the return conf in https://github.com/torproject/sbws/pull/306#discussion_r239874182 is applied first